package/MDAnalysis/converters/OpenMM.py

```diff
@@ -111,7 +111,10 @@ def _read_first_frame(self):
 
     def _mda_timestep_from_omm_context(self):
         """ Construct Timestep object from OpenMM context """
-        import simtk.unit as u
+        try:
+            import openmm.unit as u
+        except ImportError:
+            import simtk.unit as u
```

Comment:
We may want to ignore the line for coverage. Except if we have a minimum requirement CI job, in such case we need to pin OpenMM there.

Reply:
I can add a pragma no cover, it's just easier for now.